eogrow/utils/testing.py

```diff
@@ -172,26 +174,29 @@ def _calculate_tiff_stats(tiff_filename: str, config: StatCalcConfig) -> JsonDic
         }
 
 
-def _calculate_vector_stats(dataframe: gpd.GeoDataFrame, config: StatCalcConfig) -> JsonDict:
+def _calculate_vector_stats(gdf: gpd.GeoDataFrame, config: StatCalcConfig) -> JsonDict:
     """Calculates statistics over a vector GeoDataFrame"""  # TODO: add more statistical properties
 
     def _rounder(x: float, y: float) -> tuple[float, float]:
         return round(x, config.decimals), round(y, config.decimals)
 
-    dataframe.geometry = dataframe.geometry.apply(lambda geometry: shapely.ops.transform(_rounder, geometry))
+    gdf.geometry = gdf.geometry.apply(lambda geometry: shapely.ops.transform(_rounder, geometry))
 
     stats = {
-        "columns": list(dataframe),
-        "row_count": len(dataframe.index),
-        "crs": str(dataframe.crs),
-        "mean_area": _prepare_value(dataframe.area.mean(), config),
-        "total_bounds": list(dataframe.total_bounds),
+        "columns": list(gdf),
+        "row_count": len(gdf),
+        "crs": str(gdf.crs),
+        "mean_area": _prepare_value(gdf.area.mean(), config),
+        "total_bounds": list(gdf.total_bounds),
     }
 
-    if len(dataframe.index):
-        rng = np.random.default_rng(0)
-        random_rows = np.unique(rng.integers(0, len(dataframe), config.num_random_values))
-        stats["random_rows"] = {str(row): list(map(str, dataframe.iloc[row])) for row in random_rows}
+    if len(gdf):
+        subsample: gpd.GeoDataFrame = gdf.sample(min(len(gdf), config.num_random_values), random_state=42)
```

Comment:
Could also be 

```python
subsample: gpd.GeoDataFrame = gdf.sample(config.num_random_values, replace=True, random_state=42)
```

meaning that the same row can be selected twice, in case of the amount of rows being too small, but I guess it's not as important anymore

Reply:
yeah but I do not want to have row repetitions in the stats file :thinking: 
+